docs/UserGuide.md

```diff
@@ -365,7 +469,6 @@ You can manage the appointments through different commands, which can be seen in
 | **[Find appointment](#finding-appointments-by-name-and-date-find-appt)** | `find appt KEYWORD [MORE_KEYWORDS]`                                                                                      | `find appt d/2024-12-05`                                              |
 | **[Delete appointment](#deleting-an-appointment-delete-appt)**           | `delete appt INDEX`                                                                                                      | `delete appt 2`                                                       |
 | **[Clear all appointments](#clearing-all-appointments-clear-appt)**      | `clear appt`                                                                     <br/>                                   | `clear appt`                                                          |
----
 
 <br>
 
```

Comment:
Slight formatting error, for `find appt1`, can we add a `<br><br>` at the end of line 541? This is to ensure that there is a line break before the screenshot, similar to the `find person` example.